milkbaby/multisig/pubkeys/CryptoCrew.txt

```diff
@@ -0,0 +1,14 @@
+- address: bbn18ztjqfmu3rck2rv6ddwk5wuq86zfxqqqs3830r
+  name: staker
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Avy/zQFMjjafAZg5GgXdtLsZzhHNvpcxmlRyvtNzuWca"}'
+  type: ledger
+
+- address: bbn1wp36esqunthvrf2wuml07uuwag8vtu2urlmml8
+  name: staker-controller
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A88gTPlHoVzBSCG1dFc3+lCH5+FQt0/s50V4UwnHXOuY"}'
+  type: ledger
+
+- address: bbn1l924vkypf847nxtz5puf5hfll9cvrufxfnyh8t
+  name: reards-collector
```

Comment:
I'll take care of changing the key name in `main` branch. Change the key name on your side please, in order to use it to sign future transactions.

```suggestion
  name: rewards-collector
```